pkg/kubelet/token/token_manager.go

```diff
@@ -175,11 +174,15 @@ func (m *Manager) requiresRefresh(tr *authenticationv1.TokenRequest) bool {
 		klog.ErrorS(nil, "Expiration seconds was nil for token request", "tokenRequest", cpy)
 		return false
 	}
+
 	now := m.clock.Now()
 	exp := tr.Status.ExpirationTimestamp.Time
 	iat := exp.Add(-1 * time.Duration(*tr.Spec.ExpirationSeconds) * time.Second)
 
-	jitter := time.Duration(rand.Float64()*maxJitter.Seconds()) * time.Second
+	// Dynamically set maxJitter to 15% of the token's TTL
+	maxJitter := 15 / 100 * float64(*tr.Spec.ExpirationSeconds)
```

Comment:
Sorry, My fault. I mess up the type inference: Go assert `15` and `100` as integer and leads `15/100` always equal to `0`. 

```suggestion
	maxJitter := float64(*tr.Spec.ExpirationSeconds) * 15 / 100
```